tests/nlu/selectors/test_selectors.py

```diff
@@ -752,34 +910,57 @@ async def test_sparse_feature_sizes_decreased_incremental_training(
             "min_ngram": 1,
             "max_ngram": 4,
         },
-        {"name": "ResponseSelector", EPOCHS: 1},
     ]
-    _config = RasaNLUModelConfig({"pipeline": pipeline, "language": "en"})
-
-    (_, trained, persisted_path) = await rasa.nlu.train.train(
-        _config, path=str(tmpdir), data=iter1_path, component_builder=component_builder,
-    )
-    assert trained.pipeline
-
-    loaded = Interpreter.load(persisted_path, component_builder, new_config=_config,)
-    assert loaded.pipeline
-    assert loaded.parse("Rasa is great!") == trained.parse("Rasa is great!")
-    if should_raise_exception:
-        with pytest.raises(Exception) as exec_info:
-            (_, trained, _) = await rasa.nlu.train.train(
-                _config,
-                path=str(tmpdir),
-                data=iter2_path,
-                component_builder=component_builder,
-                model_to_finetune=loaded,
-            )
-        assert "Sparse feature sizes have decreased" in str(exec_info.value)
-    else:
-        (_, trained, _) = await rasa.nlu.train.train(
-            _config,
-            path=str(tmpdir),
-            data=iter2_path,
-            component_builder=component_builder,
-            model_to_finetune=loaded,
```

Comment:
this triggers a training with a model to finetune. So you need to set `is_finetuning=True` for the response selector (I think you need to use the `loaded_selector` in the outcommented parts)